7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>Is each sample a cell here, so each of the rows correspond to a sample, and the same sample would match in both the morphology data and the nomic data?</p><p> </p><p>Also, what are the canonical variables in this case (i.e. how are they defined)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>each sample is a well in this case. I updated the notebook with documentation to match.</p><p> </p><p>The canonical variables are defined later. I also added documentation to reflect this</p>